forest-node/ansible/ansible.cfg

```diff
@@ -0,0 +1,8 @@
+[defaults]
+host_key_checking = false
+inventory = hosts
+
+remote_user = root
+private_key_file = ~/.ssh/id_rsa    # assuming the name of the private key file is `id_rsa`
```

Comment:
No, I think this is an over-engineering process, we need ansible to access the key file in the user's directory. Every default key creation process stores in this directory, but if the keys are located elsewhere for any reason, the user needs to update the directory there as the case applies.

Reply:
Why not use an authentication agent instead? Pretty sure ansible supports that natively.

Reply:
Auth agent? I am not exactly clear on that - do you mean the likes of OKTA, teleport and co?

Reply:
Like [ssh-agent](https://www.ssh.com/academy/ssh/agent). I'm pretty sure you're already using it. For example, when you push commits to this repository, you don't have to unlock your private every time. The ssh agent takes care of it for you.

Reply:
I am still looking at ssh-agent tho, but it requires some manual process before ansible takes over which I am most definitely trying to avoid cos' I am moving from terraform to ansible at once. 

For me to set up ssh-agent, I need to access the server which will still require Ansible in our case and the whole purpose of `ansible.cfg` is to even initiate the login and connection process between Ansible and the server. 

I am still trying to dig but at first glance, this agent may not be able to do this perfectly.